From 36b4557daaa8ad3d8df207e3181ec38ceec3d6ec Mon Sep 17 00:00:00 2001
From: Antonio Alvarez Feijoo <antonio.feijoo@suse.com>
Date: Thu, 9 Feb 2023 13:55:11 +0100
Subject: [PATCH] fix(systemd-journald): do not include
 systemd-journal-flush.service

The systemd-journal-flush.service unit is configured not to run in the initrd
since https://github.com/systemd/systemd/commit/fe7f113c
---
 modules.d/01systemd-journald/module-setup.sh | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/modules.d/01systemd-journald/module-setup.sh b/modules.d/01systemd-journald/module-setup.sh
index 1580bdf9..dd1b3f3b 100755
--- a/modules.d/01systemd-journald/module-setup.sh
+++ b/modules.d/01systemd-journald/module-setup.sh
@@ -40,7 +40,6 @@ install() {
         "$systemdsystemunitdir"/systemd-journald-audit.socket \
         "$systemdsystemunitdir"/systemd-journald-dev-log.socket \
         "$systemdsystemunitdir"/systemd-journald-varlink@.socket \
-        "$systemdsystemunitdir"/systemd-journal-flush.service \
         "$systemdsystemunitdir"/systemd-journal-catalog-update.service \
         "$systemdsystemunitdir"/sockets.target.wants/systemd-journald-audit.socket \
         "$systemdsystemunitdir"/sockets.target.wants/systemd-journald-dev-log.socket \
@@ -63,8 +62,6 @@ install() {
             "$systemdutilconfdir/journald.conf.d/*.conf" \
             "$systemdsystemconfdir"/systemd-journald.service \
             "$systemdsystemconfdir/systemd-journald.service.d/*.conf" \
-            "$systemdsystemconfdir"/systemd-journal-flush.service \
-            "$systemdsystemconfdir/systemd-journal-flush.service.d/*.conf" \
             "$systemdsystemconfdir"/systemd-journal-catalog-update.service \
             "$systemdsystemconfdir/systemd-journal-catalog-update.service.d/*.conf" \
             "$sysusersconfdir"/systemd-journal.conf
-- 
2.43.0

